pexec/managed_process.go

```diff
@@ -127,6 +127,12 @@ func (p *managedProcess) Start(ctx context.Context) error {
 	default:
 	}
 
+	if p.cwd != "" {
+		if _, err := os.Stat(p.cwd); err != nil {
```

Comment:
I think we want this to be `Lstat` rather than stat. Such that an existing, but broken symlink generates an error. But this is just in the name of better error messaging, so definitely not necesssary.

Reply:
it's a small enough change, happy to make a flyby improvement